### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by trymeouteh** [audiamus/AaxAudioConverter#80] [General]
Please release this for Linux. Preferreabily as an AppImage that can be updated automatically just like YouTube Downloader GUI is.

https://github.com/jely2002/youtube-dl-gui

**Comment by audiamus** [comment]
That won't happen, I'm afraid. It's a Windows application by design and integrated into the Windows eco system.

Please place comments in the discussion section.

**Comment by colvdv** [comment]
### [GUIDE] Book Lib Connect/AAX Audio Converter on Linux (WINE)

**Tested and working for Book Lib Connect v0.13.1 and AAX Audio Converter v1.18.2 on Fedora 41 using WINE 10.4.**

1. Download installers:
	- [Book Lib Connect v0.13.1](https://github.com/audiamus/BookLibConnect/releases/tag/v0.13.1)
	- [AAX Audio Converter v1.18.2](https://github.com/audiamus/AaxAudioConverter/releases/tag/v1.18.2)
	- [.NET SDK 6.0.428 Windows x64](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.428-windows-x64-installer)

2. Install using Wine:
	a. Open terminal
	b. cd to the download location: `cd ~/Downloads`
	c. Run `wine dotnet-sdk-6.0.428-win-x64.exe`, follow the prompts in the installer window.
	d. Run `wine BookLibConnect-0.13.1-Setup.exe`, follow the prompts and use the recommended settings.
	e. Run `wine AaxAudioConverter-1.18.2-Setup.exe`, follow the prompts and use the recommended settings.

5. Launch Book Lib Connect and follow the prompts to configure it. If met with any unhandled exceptions, just click continue until they go away. Do the same for AAX Audio Converter.

7. You should now be able to use Book Lib Connect and AAX Audio Converter on Linux!

_Related Discussion:_ _https://github.com/audiamus/BookLibConnect/discussions/196_

**Reply by glindeman** [reply]
Has anyone tried this for Mac using Crossover?

It worked great for me a few months ago, but now I've tried just the aax audio converter install without any luck and also followed these instructions without any luck.

BookLibConnect is working when I follow this, but when I try to open aax audio converter it just closes before any window opens up

**Reply by glindeman** [reply]
I found the solution. Downloading the .net 4.8 framework from crossover resolves this issue
